platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/StateConfig.java

```diff
@@ -28,7 +28,7 @@
  *                                      directory, unless the provided path begins with "/", in which case it will be
  *                                      interpreted as an absolute path.
  * @param mainClassNameOverride         Typically, the fully qualified name of the application class implementing
- *                                      {@link com.swirlds.common.system.SwirldMain SwirldMain} is used as a directory
+ *                                      SwirldMain is used as a directory
```

Comment:
it should be  {@link com.swirlds.platform.system.SwirldMain}

Reply:
This link can no longer exist without adding a circular dependency on swirlds common and swirlds core.

Eventually I'd like to move this configuration out of swirlds common, but since I could leave it out of the big PR with only javadoc changes I decided to wait until later.